workers/common/src/test/kotlin/common/OrtRunServiceTest.kt

```diff
@@ -212,7 +254,7 @@ class OrtRunServiceTest : WordSpec({
                     severity = Severity.WARNING,
                     identifier = identifier1,
                     worker = "scanner"
-                    ),
+                ),
```

Comment:
Nit: Unrelated formatting fix that could be ext6ract to a preceding commit.

Reply:
Dropped this change.